website_crm_mandatory_phone_number/views/website_crm.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="website_crm_mandatory_phone_number_template" name="Phone Number Mandatory"
+                  inherit_id="website_crm.contactus_form" customize_show="True">
+            <xpath expr="//input[@name='phone']" position="attributes">
+                <attribute name="required">True</attribute>
```

Comment:
as of http://www.w3.org/TR/2008/WD-html5-20080610/semantics.html 3.2.2, this should be `required` and not `True`